internal/executor/configuration/podchecks/types.go

```diff
@@ -26,8 +26,9 @@ const (
 )
 
 type Checks struct {
-	Events            []EventCheck
-	ContainerStatuses []ContainerStatusCheck
+	Events                    []EventCheck
+	ContainerStatuses         []ContainerStatusCheck
+	TimeWithoutEventsOrStatus time.Duration
```

Comment:
'TimeWithoutEventsOrStatus' might be clearer as 'DeadlineForUpdates'

Reply:
Done.